integration_tests/src/main/python/arithmetic_ops_test.py

```diff
@@ -198,13 +198,50 @@ def test_multiplication_ansi_enabled(data_gen):
                 f.col('a') * f.col('b')),
             conf=ansi_enabled_conf)
 
-def test_multiplication_ansi_overflow():
+def test_multiplication_ansi_overflow_for_decimal():
     exception_str = 'ArithmeticException'
     assert_gpu_and_cpu_error(
         lambda spark : unary_op_df(spark, DecimalGen(38, 0)).selectExpr("a * " + "9"*38 + " as ret").collect(),
         ansi_enabled_conf,
         exception_str)
 
+_gen_gen_lit_tuple_list_overflow_for_multiply = [
+    (ByteGen(min_val=30, max_val=50, special_cases=[]), ByteGen(min_val=50, max_val=127, special_cases=[]), -50),  # 30 * 50 > 127 (Byte.MaxValue)
+    (ShortGen(min_val=-32768, max_val=-30000, special_cases=[]), ShortGen(min_val=2, max_val=32767, special_cases=[]), 2), # 30000 * 2 > 32767 (Short.MaxValue)
+    (IntegerGen(min_val=2147483647//2, max_val=2147483647, special_cases=[]), IntegerGen(min_val=3, max_val=2147483647, special_cases=[]), 3) , # 2147483647//2 * 3 > 2147483647 (Int.MaxValue)
+    (LongGen(min_val=9223372036854775807//2, max_val=9223372036854775807, special_cases=[]), LongGen(min_val=3, max_val=9223372036854775807, special_cases=[]), 2147483647)] # 9223372036854775807//2 * 3 > 9223372036854775807(Long.MaxValue)
```

Comment:
We have the max values defined in data_gen.py for byte, short, int and long, e.g. `LONG_MAX`, `INT_MAX`, better to use them instead.